packages/itwin/breakdown-trees/package.json

```diff
@@ -95,38 +96,34 @@
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
     "rimraf": "^3.0.2",
-    "tslint": "^5.20.0",
-    "typescript": "~4.3.0",
+    "typescript": "^4.4.0",
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
-    "@bentley/bentleyjs-core": "^2.19.9",
-    "@bentley/geometry-core": "^2.19.9",
-    "@bentley/imodelhub-client": "^2.19.9",
-    "@bentley/imodeljs-common": "^2.19.9",
-    "@bentley/imodeljs-frontend": "^2.19.9",
-    "@bentley/imodeljs-i18n": "^2.19.9",
-    "@bentley/imodeljs-markup": "^2.19.9",
-    "@bentley/imodeljs-quantity": "^2.19.9",
-    "@bentley/itwin-client": "^2.19.9",
-    "@bentley/presentation-common": "^2.19.9",
-    "@bentley/presentation-components": "^2.19.9",
-    "@bentley/presentation-frontend": "^2.19.9",
-    "@bentley/product-settings-client": "^2.19.9",
-    "@bentley/ui-abstract": "^2.19.9",
-    "@bentley/ui-components": "^2.19.9",
-    "@bentley/ui-core": "^2.19.9",
-    "@bentley/ui-framework": "^2.19.9",
-    "@bentley/ui-ninezone": "^2.19.9",
-    "@bentley/webgl-compatibility": "^2.19.9",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
+    "@itwin/core-i18n": "^3.0.0",
+    "@itwin/core-markup": "^3.0.0",
+    "@itwin/core-quantity": "^3.0.0",
+    "@itwin/presentation-common": "^3.0.0",
+    "@itwin/presentation-components": "^3.0.0",
+    "@itwin/presentation-frontend": "^3.0.0",
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/appui-layout-react": "^3.0.0",
+    "@itwin/webgl-compatibility": "^3.0.0",
     "@bentley/webpack-tools": "1.14.1",
-    "@bentley/measure-tools-react": "^0.5.0",
+    "@itwin/measure-tools-react": "workspace:^0.2.0",
```

Comment:
i _think_ you can do `workspace:^*`

Reply:
could you try that out and lmk if that works?

Reply:
Yes, this works and we had it before.. But in my last PR you suggested to use it this way in peer deps as the iModel repo does.. So I changed it to this.